react-native-template-pytorch-live/script.js

```diff
@@ -9,17 +9,29 @@
  */
 
 const execSync = require('child_process').execSync;
+const process = require('process');
+
+const makeModels = () => {
+  execSync('source ./venv/bin/activate && \
+              python3 -m pip install --upgrade pip && \
+              python3 -m pip install -r requirements.txt --no-cache-dir && \
+              python3 -W ignore make_models.py');
+}
+
+process.chdir('./models');
 
 switch (process.platform) {
   case 'darwin':
+    execSync('brew install python@3.9');
+    execSync('/usr/local/opt/python@3.9/bin/python3 -m venv ./venv');
```

Comment:
The absolute path can vary depending on how Homebrew is set up.  On my machine, for example, Homebrew installs packages in my user's home directory:

```
$HOME/homebrew/opt/python@3.9
```